httpclient5/pom.xml

```diff
@@ -105,6 +105,7 @@
         <directory>src/main/resources</directory>
         <filtering>true</filtering>
         <includes>
+            <include>**/*.dat</include>
```

Comment:
@garydgregory Do we really need to filter `.dat` files? Seems unnecessary to me,

Reply:
Hello @ok2c 
Yes, because other resources are already filtered and we want to include the new `.dat` file.

Alternatives are:
- Remove all `<include>` filters, but add an `<exclude>` filter for the `readme.txt` that explains the `.dat` file.
- Remove the whole `<resources>` section from that `pom.xml`, which means the including `readme.txt` that explains the `.dat` file.
- Remove all `<include>` filters, remove the `readme.txt`, which let's you then remove the whole `<resources>` section from that `pom.xml`
- Keep the PR as is.

Do you have a preference?

Reply:
> Yes, because other resources are already filtered and we want to include the new `.dat` file.

@garydgregory I may be mistaken, but resource filtering is used to replace variable placeholders with system or project variables, not to filter (include / exclude) resources. [1]. We already have `mozilla/public-suffix-list.txt` resource that gets added to the artifact jar, even though `*.txt` files are not filters.  Am I missing something?

We do not need to filter `.dat` files, do we? 

[1] https://maven.apache.org/plugins/maven-resources-plugin/examples/filter.html

Reply:
Hello @ok2c 
Well, we're both right. I've pushed a new version that cleanly separates the concerns of which file to copy and whether to filter it or not by using two `<resource>` elements:

- We want the `.properties` file copied _and_ filtered. The git master version causes _only_ the properties file to be copied, the `.dat` file is not copied.
- We want to the `.dat` file copied but _not_ filtered.